### DISCUSSION TITLE
How did you build your handlebar files?

### DISCUSSION THREAD

**Discussion by dante01yoon** [dani-garcia/vaultwarden#3142] [Q&A]
Greetings, I looked into this projects, and couldn't find configs to build handlebars into raw html static files.

How do you achieve this? do you use grunt? or any other build tools to transform to html from template?

**Comment by BlackDex** [comment]
It's done in code once during build, and if there is a template directory provided once during runtime, unless the always refresh flag is set.

**Reply by dante01yoon** [reply]
so what build  tool do  you use to transform template to html?

**Reply by BlackDex** [reply]
Its in code. There is no build tool.
There is a create which does all in code as mentioned before.
https://crates.io/crates/handlebars

**Reply by dante01yoon** [reply]
I see. thank you!
